src/dotty/tools/dotc/transform/LazyVals.scala

```diff
@@ -105,17 +100,19 @@ class LazyVals extends MiniPhaseTransform with SymTransformer {
 
         val holderSymbol = ctx.newSymbol(x.symbol.owner, holderName, containerFlags, holderImpl.typeRef, coord = x.pos)
         val initSymbol = ctx.newSymbol(x.symbol.owner, initName, initFlags, MethodType(Nil, tpe), coord = x.pos)
-        val result = ref(holderSymbol).select("value".toTermName)
-        val flag = ref(holderSymbol).select("initialized".toTermName)
+        val result = ref(holderSymbol).select(nme_value)
```

Comment:
Non-standard naming convention: `nme_...` should be avoided. See my comment to `object LazyVals`.